lxc/utils.go

```diff
@@ -315,3 +316,72 @@ func getServerSupportedFilters(filters []string, i interface{}) ([]string, []str
 
 	return supportedFilters, unsupportedFilters
 }
+
+func guessImage(conf *config.Config, d lxd.InstanceServer, remote string, iremote string, image string) (string, string) {
```

Comment:
Please can you separate these into separate commit(s) independent of instance rebuild logic?